src/assets/stylesheets/screens/_sign-in.scss

```diff
@@ -0,0 +1,26 @@
+.container-sign-in {
+  display: flex;
+  flex-direction: column;
+  width: rem(320px);
+}
+
+.form-input-group {
+  display: flex;
+  flex-direction: column;
+  width: rem(320px);
+  height: rem(100px);
```

Comment:
@gutakk The height doesn't seem right 🤔 
![Screen Shot 2564-06-18 at 19 31 03](https://user-images.githubusercontent.com/1772999/122561203-c7607e80-d06b-11eb-8818-c1b5d0502c81.png)
How about remove height here and add the margin instead?

```scss
&:not(:first-child) {
    margin-top: rem(24px);
  }
```

![Screen Shot 2564-06-18 at 19 34 02](https://user-images.githubusercontent.com/1772999/122561545-30e08d00-d06c-11eb-820c-fc7c5a90169f.png)

Reply:
That would be nice. Fixed in 604ea54c478ca03356a7826207d54d3302077de7 ddb6bad20e3f656e2d005cd6255597a377b4aeab